File: app/src/main/java/com/moictab/curriculumvitae/model/Project.java
package com.moictab.curriculumvitae.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by moict on 16/10/2017.
 */

public class Project {
    @SerializedName("title")
    public String title;

    @SerializedName("period")
    public String period;

    @SerializedName("description")
    public String description;
}

File: app/src/main/java/com/moictab/curriculumvitae/model/Project.kt
package com.moictab.curriculumvitae.model

import com.google.gson.annotations.SerializedName

/**
 * Created by moict on 16/10/2017.
 */

class Project {
    @SerializedName("title")
    var title: String? = null

    @SerializedName("period")
    var period: String? = null

    @SerializedName("description")
    var description: String? = null
}
